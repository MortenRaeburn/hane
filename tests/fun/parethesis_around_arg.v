Definition bar : forall foo : Type{99}, Type{99} := fun (bar : Type{99}) => bar.