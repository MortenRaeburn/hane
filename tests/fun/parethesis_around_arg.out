Definition bar : forall[Type{99}] (Type{99}) := fun[Type{99}] ('0).
